beetmoverscript/docker.d/worker.yml

```diff
@@ -93,6 +102,15 @@ clouds:
                 name: 'mozilla'
                 location: 'us'
                 project: 'moz-fx-productdelivery-pr-38b5'
+            product_yum_repos:
+              devedition:
+                name: 'mozilla-rpm'
+                location: 'us'
+                project: 'moz-fx-productdelivery-no-38b5'
```

Comment:
Seems wrong for prod to push to the stage repo, so if this is intended it needs some kind of explanation.

Reply:
Nope, just me copy pasting, not realizing that no/pr were also different...